libraries/classes/Controllers/UserPasswordController.php

```diff
@@ -62,15 +62,18 @@ public function __invoke(ServerRequest $request): void
             return;
         }
 
+        $noPass = $request->getParsedBodyParam('nopass');
+        $pmaPw = $request->getParsedBodyParam('pma_pw');
+
         /**
          * If the "change password" form has been submitted, checks for valid values
          * and submit the query or logout
          */
-        if (isset($_POST['nopass'])) {
-            if ($_POST['nopass'] == '1') {
+        if (isset($noPass])) {
```

Comment:
same, it is always isset + typo fix
```suggestion
        if ($noPass !== null) {
```